app/controllers/application_controller.rb

```diff
@@ -1402,45 +1402,68 @@ def set_config(db_record) # pass in the db record, either @host or @vm
           dev = disk.controller_type ? disk.controller_type << " " << loc : ""  # default device is controller_type
           desc = disk.filename                              # default description is filename
           icon = disk.device_name                       # default icon prefix is device_name
-          conn = disk.start_connected ? ", Connect at Power On = Yes" : ", Connect at Power On = No"
+          conn = disk.start_connected ? _(", Connect at Power On = Yes") : _(", Connect at Power On = No")
           # Customize disk entries by type
           if disk.device_type == "cdrom-raw"
-            dev = "CD-ROM (IDE " << loc << ")" << conn
+            dev = "CD-ROM (IDE %{location})%{connection}" % {:location => loc, :connection => conn}
```

Comment:
Shouldn't this string be wrapper with `_()`? Same for other strings like this below.